src/engine/platform/k007232.cpp

```diff
@@ -158,7 +158,7 @@ void DivPlatformK007232::tick(bool sysTick) {
       else {
         chan[i].lvol=chan[i].rvol=isMuted[i]?0:chan[i].outVol&0xf;
         const unsigned char prevVolume=lastVolume;
-        lastVolume=(lastVolume&~(0xf<<(i<<2)))|((chan[i].outVol&0xf)<<(i<<2));
+        lastVolume=(lastVolume&~(0xf<<(i<<2)))|((chan[i].lvol&0xf)<<(i<<2));
```

Comment:
Hmm?

Reply:
Because common mono setup is using SLEV output as volume register, and it's packed as nibble; Low nibble determines volume of first channel and High nibble determines second channel.